app/containers/Applications/ApplicationRevisionStatusContainer.tsx

```diff
@@ -103,78 +103,59 @@ export const ApplicationRevisionStatusComponent: React.FunctionComponent<Props>
 
   return (
     <>
-      <Row>
-        <Col md={3}>
```

Comment:
Eventually this component will be removed once "make a decision" is implemented in GGIRCS-2294. Until then, this just removes the `Row` and `Col` layout wrappers to make it inline with the page heading.